kythe/docs/schema/schema.txt

```diff
@@ -501,8 +501,8 @@ lines owing to heuristics in Clang's comment parser.
 
 [kythe,C++,"Comments document objects."]
 --------------------------------------------------------------------------------
-int v;   //- @"/// An empty class." documents ClassC
-         //- ClassC.node/kind record
+//- @"/// An empty class." documents ClassC
+//- ClassC.node/kind record
```

Comment:
I'm not entirely sure why, but apparently this change causes the schema tests to fail, @zrlk ?

Reply:
My guess, based on re-re-re-reading the text above the example, is that the original is a way of getting the two sets of comments to be separate (the `//-` comments for the verifier and the `///` comment for the test). So, I should remove my PR and leave future generations to puzzle this out. ;)

Reply:
Without the `int v;` there, I think Clang will merge these comments with the class comment into a single block.